src/app/news/news-list-item.component.ts

```diff
@@ -20,6 +20,37 @@ export class NewsListItemComponent {
     private userService: UserService
   ) {}
 
+  ngOnInit() {
+    this.setDisplayMessage(this.item);
+  }
+
+  setDisplayMessage(news) {
+    const lines: string[] = news.message.split('\n\n');
```

Comment:
The message may not have newline characters to denote each line.  A long paragraph could be hundreds of lines because the words wrap around without a newline character.

You should set a max height when the message is in showLess mode and there is overflow.  I'm not sure off the top of my head how to detect for overflow, but maybe you can find that with a little research.